nomad/operator_endpoint.go

```diff
@@ -284,3 +284,64 @@ func (op *Operator) ServerHealth(args *structs.GenericRequest, reply *autopilot.
 
 	return nil
 }
+
+// SchedulerSetConfiguration is used to set the current Scheduler configuration.
+func (op *Operator) SchedulerSetConfiguration(args *structs.SchedulerSetConfigRequest, reply *bool) error {
```

Comment:
Why does the autopilotset endpoint do the same thing? I figured bool was fine because its basically only ever going to be true/false for the compare and set case.

Reply:
I would still change this. We have talked about the downside of not having flexibility on the endpoints pretty extensively